lib/parity/backup.rb

```diff
@@ -77,21 +93,34 @@ def ensure_temp_directory_exists
     end
 
     def download_remote_backup
-      Kernel.system(
-        "curl -o tmp/latest.backup \"$(heroku pg:backups:url --remote #{from})\"",
-      )
+      if backup_id
+        puts "Downloading backup #{backup_id} from #{from}..."
+        Kernel.system(
+          "curl -o tmp/#{backup_id}.backup \"$(heroku pg:backups:url #{backup_id} --remote #{from})\"",
+        )
+      else
+        puts "Downloading latest backup from #{from}..."
+        Kernel.system(
+          "curl -o tmp/latest.backup \"$(heroku pg:backups:url --remote #{from})\"",
```

Comment:
Could you use the same curl statement in both places since a nil `backup_id` would be an empty string here? This would avoid having two pieces of interpolated string arguments to maintain going forward where we could have one.